.github/readme-de_de.md

```diff
@@ -0,0 +1,383 @@
+
+## Fragen oder Vorschläge?
+
+### Discord-Server
+
+| [![][discord-shield-badge]][discord-link] | [Tritt unserer Discord-Community bei!](https://discord.gg/sillytavern) Erhalte Unterstützung, teile deine Lieblingscharaktere und Eingabeaufforderungen. |
+| :---------------------------------------- | :----------------------------------------------------------------------------------------------------------------- |
+
+Oder nimm direkt Kontakt mit den Entwicklern auf:
+
+* Discord: cohee, rossascends, wolfsblvt
+* Reddit: [/u/RossAscends](https://www.reddit.com/user/RossAscends/), [/u/sillylossy](https://www.reddit.com/user/sillylossy/), [u/Wolfsblvt](https://www.reddit.com/user/Wolfsblvt/)
+* [Erstelle ein GitHub-Problem](https://github.com/SillyTavern/SillyTavern/issues)
+
+### Ich mag dein Projekt! Wie kann ich beitragen?
+
+1. Sende Pull-Requests. Lerne, wie du beitragen kannst: [CONTRIBUTING.md](../CONTRIBUTING.md)
+2. Sende Funktionser Vorschläge und Problemmeldungen unter Verwendung der bereitgestellten Vorlagen.
+3. Lies diese gesamte README-Datei und überprüfe zuerst die Dokumentationswebsite, um doppelte Probleme zu vermeiden.
+
+## Screenshots
+
+<img width="500" alt="image" src="https://github.com/user-attachments/assets/9b5f32f0-c3b3-4102-b3f5-0e9213c0f50f">
+<img width="500" alt="image" src="https://github.com/user-attachments/assets/913fdbaa-7d33-42f1-ae2c-89dca41c53d1">
+
+## Charakterkarten
+
+SillyTavern basiert auf dem Konzept der "Charakterkarten". Eine Charakterkarte ist eine Sammlung von Eingabeaufforderungen, die das Verhalten des LLM festlegen und erforderlich sind, um anhaltende Gespräche in SillyTavern zu führen. Sie funktionieren ähnlich wie ChatGPTs GPTs oder Poes Bots. Der Inhalt einer Charakterkarte kann alles sein: ein abstraktes Szenario, ein Assistent, der für eine bestimmte Aufgabe maßgeschneidert ist, eine berühmte Persönlichkeit oder ein fiktiver Charakter.
+
+Das Namensfeld ist der einzige erforderliche Eingabewert für die Charakterkarte. Um ein neutrales Gespräch mit dem Sprachmodell zu beginnen, erstelle eine neue Karte, die einfach "Assistent" genannt wird, und lasse die restlichen Felder leer. Für einen thematischeren Chat kannst du dem Sprachmodell verschiedene Hintergrundinformationen, Verhaltensweisen und Schreibmuster sowie ein Szenario geben, um das Gespräch zu beginnen.
+
+Um ein schnelles Gespräch zu führen, ohne eine Charakterkarte auszuwählen, oder um einfach die LLM-Verbindung zu testen, gib einfach deine Eingabeaufforderung in die Eingabezeile auf dem Willkommensbildschirm ein, nachdem du SillyTavern geöffnet hast. Bitte beachte, dass solche Chats vorübergehend sind und nicht gespeichert werden.
+
+Um eine allgemeine Vorstellung davon zu bekommen, wie man Charakterkarten definiert, sieh dir die Standardfigur (Seraphina) an oder lade ausgewählte von der Community erstellte Karten im Menü "Erweiterungen & Assets herunterladen" herunter.
+
+## Hauptmerkmale
```

Comment:
```suggestion
## Wichtigste Features
```
I don't think most places actually translate "Features". *Merkmale* is not the correct wording for sure.